sqlitex/query.go

```diff
@@ -100,3 +102,16 @@ func ResultFloat(stmt *sqlite.Stmt) (float64, error) {
 	}
 	return res, nil
 }
+
+// ResultBytes reads the first column of the first and only row produced by running stmt into buf, returning the number of bytes read.
+// It returns an error if there is not exactly one result row.
+func ResultBytes(stmt *sqlite.Stmt, buf []byte) (int, error) {
```

Comment:
Mind adding a line in the CHANGELOG for this new function? (See surrounding entries for format and please link to this PR.)